roosevelt.js

```diff
@@ -111,8 +113,13 @@ module.exports = function (params) {
   app.use(require('cookie-parser')())
 
   // enable favicon support
-  if (app.get('params').favicon !== 'none') {
-    app.use(require('serve-favicon')(path.join(app.get('appDir'), app.get('params').staticsRoot, app.get('params').favicon)))
+  if (app.get('params').favicon !== 'none' || app.get('params').favicon !== null) {
+    faviconPath = path.join(app.get('appDir'), app.get('params').staticsRoot, app.get('params').favicon)
+    if (fileExists(faviconPath)) {
+      app.use(require('serve-favicon')(faviconPath))
+    } else {
+      logger.warn(`${app.get('params').favicon} specified in favicon does not exist. Please ensure file is entered properly.`.yellow)
```

Comment:
Favicon images/favicon.ico does not exist. Please ensure the "favicon" param is configured correctly.

Reply:
Also change favicon default to "none" to suppress the warning. The generator can opt-in to favicons.